recipe/meta.yaml

```diff
@@ -0,0 +1,58 @@
+{% set name = "rbcl" %}
+{% set version = "1.0.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.python.org/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: a0bf43234a2987b67b451d1e61efe5666c12a236001e8ace788307ce5f2ad698
+  patches:
+    - 0001-do-not-compile-binary.patch
+    - 0002-import-appropriate-libsodium-binary-depending-on-sys.patch
+
+build:
+  number: 0
+  script:
+    - mkdir -p $SRC_DIR/build/temp/lib  # [not win]
+    - mkdir -p %SRC_DIR%\build\temp\lib  # [win]
+    - cp $PREFIX/lib/libsodium.* $SRC_DIR/build/temp/lib  # [not win]
+    - cp %LIBRARY_BIN%\libsodium.dll %SRC_DIR%\build\temp\lib  # [win]
```

Comment:
is it really necessary, do not we have in env already ?

Reply:
it is necessary because that's the path where __sodium_build.py_ takes it from:
```
path_libsodium = 'build/temp/lib/libsodium.so' # Default for non-Windows builds.
```
`build/temp/lib/libsodium.so` refers to paths inside of `$SRC_DIR`. I would rather not have an install file take files directly from outside the build CWD, although I agree it's a questionable point of view, since I still had to patch __sodium_build.py_.

Reply:
You can take it from https://github.com/nthparty/rbcl/blob/2018e3a0059276ce674d307c111eee9d8238e814/src/rbcl/_sodium.tmpl#L31

Reply:
I can make this change anywhere...in the template, in the install files or in the script section. I prefer to do it in the script section. Is there any preference as to doing this elsewhere?

Reply:
changed.